docs/tutorial2023_unfolding/unfolding_exercise.md

```diff
@@ -142,26 +143,33 @@ Then the `shape` nuisance parameters can be defined in the systematics block in
 
 In the realistic CMS analysis there are hundreds of nuisance parameters corresponding to various source of systematics. 
 
-When we unfold to the gen-level observable we should remove the nuisances affecting the rate of the gen-level bins, i.e. the `lnN` NPs: `THU_ZH_mig*, THU_ZH_inc` and keep only the acceptance `shape` uncertainties: `THU_ZH_acc` and `THU_ggZH_acc`. This can be achieved by freezing the respective nuisance parameters with the option `--freezeParameters par_name1,par_name2`. Alternatively you can create a group following the syntax given below at the end of the combined datacard, and freeze the parameters with the `--freezeNuisanceGroups group_name` option.
+When we unfold to the gen-level observable we should remove the nuisances affecting the rate of the gen-level bins, i.e. the `lnN` NPs: `THU_ZH_mig*, THU_ZH_inc` and keep only the acceptance `shape` uncertainties: `THU_ZH_acc` and `THU_ggZH_acc`, which do not scale the inclusive cross sections by construction. 
+This can be achieved by freezing the respective nuisance parameters with the option `--freezeParameters par_name1,par_name2`. Alternatively you can create a group following the syntax given below at the end of the combined datacard, and freeze the parameters with the `--freezeNuisanceGroups group_name` option.
```

Comment:
Maybe before this it would be good to point out explicitly that for this analysis you have already done the work of separating out the normalization and shape components by normalizing the theoretical variations to the nominal generator-level cross sections for each bin?